applications/web_server/html/time_ntp.html

```diff
@@ -67,4 +80,60 @@ <h3>Time - NTP</h3>
 })
 .change();
 
+
+(function($) {
+
+    var time_ntp_form = document.getElementById('js-time_ntp-form');
+
+    time_ntp_form.addEventListener('submit', function(e) {
+    $(".status").empty();
+
+    var checkBox = document.getElementById("chkNtpEnable");
+    var chkNtpEnable = 0;
+    var txtNtpServer, txtDate, txtTime;
+    if(checkBox.checked == true)
+    {
+        chkNtpEnable = 1;
+        txtNtpServer = $('#txtNtpServerAddress').val();
+        txtTime="";
+        txtDate="";
+    }
+    else
+    {
+        chkNtpEnable = 0;
+        txtNtpServer = "";
+        txtDate = $('#txtDate').val();
+        txtTime = $('#txtTime').val();
+    }
+
+    e.preventDefault();
+
+    var formData = new FormData();
+    formData.append('enable_ntp', chkNtpEnable);
+    formData.append('ntp_server', txtNtpServer);
+    formData.append('date', txtDate);
+    formData.append('time', txtTime);
```

Comment:
Should we follow openvpn or adduser page to append data in FormData as a standard?